src/Ztm.WebApi/ControllerBaseExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static bool TryGetCallbackUrl(this ControllerBase controller, out Uri url
                 }
                 catch (UriFormatException ex)
                 {
-                    throw new InvalidCallbackUrlException(ex);
+                    throw new InvalidCallbackUrlException(null, ex);
```

Comment:
We should provide exception message so we know what happening. Even though it not used to response to client but it will be helpful in our log.